src/seedu/addressbook/data/person/Address.java

```diff
@@ -35,24 +31,11 @@ public static boolean isValidAddress(String test) {
         return test.matches(ADDRESS_VALIDATION_REGEX);
     }
 
-    @Override
-    public String toString() {
-        return value;
-    }
 
     @Override
     public boolean equals(Object other) {
```

Comment:
Usually `equals(...)` and `hashCode()` are working together, so it is recommended that you abstract both of them or not at the same time.